x/incentive/keeper/scenario_test.go

```diff
@@ -182,3 +182,44 @@ func TestBasicIncentivePrograms(t *testing.T) {
 		"bob pending rewards at time 300",
 	)
 }
+
+func TestZeroBonded(t *testing.T) {
+	const (
+		umee  = fixtures.UmeeDenom
+		uumee = leveragetypes.UTokenPrefix + fixtures.UmeeDenom
+	)
```

Comment:
let's make a global const - it will be used few times.

Reply:
Also let's rename `uumee` to something else (maybe `lumee` or `ut_umee`) to not confuse with `uumee` base denom.